deploy/examples/cluster-multus-test.yaml

```
@@ -28,6 +28,12 @@ spec:
     selectors:
       public: public-net
       cluster: cluster-net
+    addressRanges:
+      public:
+        - "192.168.20.0/30" # used for pods via NetAttDef
```

```suggestion
        - "192.168.20.0/30" # used for pods via NetAttDef
```

after adding this...

mons pods are not coming up 


and multus validation test also fails with
```
RESULT: multus validation test failed: multus validation test timed out: no web server network info: pod has no network status yet: cannot find network status
```

Seems like IP's are not available.